src/renderer/shared/lib/utils/substrate.ts

```diff
@@ -44,7 +45,8 @@ export const createTxMetadata = async (
     blockNumber: block.header.number.toNumber(),
     genesisHash: genesisHash.toString(),
     metadataRpc: metadataRpc.toHex(),
-    nonce: nonce.toNumber(),
+    nonce: nonce.toNumber() + nonceDelta,
```

Comment:
Why `nonceDelta` was required to be added? Just curious 😌

Reply:
If we execute several txs from one account, we have to upgrade nonce for all accounts, added it to function arguments comments